src/main/java/seedu/address/model/util/SerializeUtil.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.util;
+
+
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ListEntryField;
+
+/**
+ * A utility class for serializing and deserializing ListEntryFields.
+ */
+public class SerializeUtil {
+    /**
+     * Serializes a ListEntryField into a string.
+     */
+    public static String serialize(ListEntryField obj) {
+        return obj.toString();
```

Comment:
Would it be better to use a method like `obj.serialize()` instead? Reasoning: Sometimes the .toString() be used in other places where it might not be used for serialization

Reply:
That is why I restrict the input type to ListEntryField
Or add .serialize and .deserialize to all object instead if you like

Reply:
And last point, this is just a util class to reduce repeated code and simply logic, it is not intrusive. You are free to not use it to generate serialized string anyway.

Reply:
Ok agreed on your points